nextflow_schema.json

```
@@ -155,6 +155,25 @@
                 }
             }
         },
+        "reads_qc": {
+            "title": "Reads QC options",
+            "type": "object",
+            "fa_icon": "fab fa-acquisitions-incorporated",
+            "description": "Set the thresholds for the reads QC/filtering steps. Reads that fail QC won't be assembled.",
+            "help_text": "Use these options to define the quality control thresholds for your reads. You can specify the maximum allowed filtering ratio and the minimum acceptable read count. If the filtering ratio exceeds the set limit or the read count falls below the threshold, the reads will be flagged and excluded from further assembly. The information about those runs that failed are aggregated in the qc_failed_runs.csv file.",
+            "properties": {
+                "filter_ratio_threshold": {
+                    "type": "number",
+                    "description": "The maximum allowed ratio of reads after filtering compared to before filtering. If exceeded, it flags excessive filtering. The default value is 0.9, meaning that if more than 90% of the reads are filtered out, the threshold is considered exceeded and the run is not assembled.",
+                    "default": 0.9
```

Make bounds clear that this is a fractional ratio
```suggestion
                    "default": 0.9
                    "minimum": 0.0,
                    "maximum": 1.0
```